meshroom/nodes/aliceVision/KeyframeSelection.py

```diff
@@ -168,6 +88,13 @@ class KeyframeSelection(desc.CommandLineNode):
             range=(0, 10000, 1),
             uid=[0],
         ),
+        desc.BoolParam(
+            name='useSimpleMethod',
+            label='Use simple method',
```

Comment:
Upper Camel Case for Labels as everywhere.

Reply:
Done.